sympy/integrals/integrals.py

```diff
@@ -468,6 +469,13 @@ def doit(self, **hints):
         reps = {}
         for xab in self.limits:
             if len(xab) != 3:
+                # it makes since to just make
```

Comment:
sense?

This happens (sort of) a number of lines down and there is a "better" approach in #21985 (in the sense that the Dummy-symbols will inherit properties from the bounds). However, that is only for definite integrals.